xml/schema/types/layoutblocks-2-9-6.xsd

```diff
@@ -38,7 +38,7 @@
         <xs:element name="layoutblock" minOccurs="0" maxOccurs="unbounded">
           <xs:complexType>
             <xs:sequence>
-              <xs:element name="systemName" type="systemNameType" minOccurs="1" maxOccurs="1"/>
+              <xs:element name="systemName" type="systemNameType" minOccurs="0" maxOccurs="1"/>
```

Comment:
Our XML Schema "allow" a lot of things.  The 2.9.6 version needs to validate 
- elements with just the systemName attribute
- elements with just the systemName subelement
- elements with both
I don't know of a clean way to validate those, and not validate when neither are present.  If you do, that would be great to add.

In the long run, I'd like to finish this migration by (in order)
- have the `store(..)` methods stop writing the attribute form for systemName and userName
- then have the files stored pointing to a newer schema version that won't accept the attribute form and insists on the systemName element